lms/product/canvas/_plugin/misc.py

```diff
@@ -24,7 +24,7 @@ def post_launch_assignment_hook(self, request, js_config, assignment):
         # `lis_result_sourcedid` associates a specific user with an
         # assignment.
         if (
-            assignment.is_gradable
+            self.is_assignment_gradable(lti_params)
```

Comment:
Trust the current launch info instead of the DB.

While this is not gradable is because SpeedGrader, so I reckon this is the right fix.

assignment.is_gradable should be True, because this is a gradable assignment but we  shouldn't include any grading functionality in this launch because Canvas isn't sending it.